src/main/java/com/thealgorithms/datastructures/trees/LevelOrderTraversal.java

```diff
@@ -1,58 +1,46 @@
 package com.thealgorithms.datastructures.trees;
 
-public class LevelOrderTraversal {
-
-    class Node {
-
-        int data;
-        Node left, right;
-
-        public Node(int item) {
-            data = item;
-            left = right = null;
-        }
-    }
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
 
-    // Root of the Binary Tree
-    Node root;
-
-    public LevelOrderTraversal(Node root) {
-        this.root = root;
-    }
-
-    /* function to print level order traversal of tree*/
-    void printLevelOrder() {
-        int h = height(root);
-        int i;
-        for (i = 1; i <= h; i++) {
-            printGivenLevel(root, i);
-        }
-    }
+/* Class to print Level Order Traversal */
+public class LevelOrderTraversal {
 
-    /* Compute the "height" of a tree -- the number of
-  nodes along the longest path from the root node
-  down to the farthest leaf node.*/
-    int height(Node root) {
+    /* Given a binary tree. Print its nodes in level order
+  using array for implementing queue  */
+    static List<List<Integer>> traverse(BinaryTree.Node root) {
```

Comment:
Traversal is just a sequence of nodes, why does it return `List<List<Integer>>` and not `List<Integer>`?

Reply:
Actually, it's not just a sequence of nodes. Here we return ```List<List<Integer>>``` to store nodes level by level. Also, this approach can help to print the nodes level by level.